include/RAJA/util/LocalArray.hpp

```diff
@@ -41,7 +41,7 @@ namespace RAJA
 
 
 template<camp::idx_t ... Sizes>
-using ParamList = camp::idx_seq<Sizes...>;
+using InitList = camp::idx_seq<Sizes...>;
```

Comment:
Can you explain the need for InitList?
The LocalArray's are indeed Params, so wouldn't ParamList be appropriate for InitLocalMem?
Or was there some disambiguation reason?

Reply:
There was some ambiguity issues, I developed a Paramlist for the new kernel interface but its different than the existing version. Thanks for the comments! I'll address these and do some clean up.